.tekton/pipeline/acceptance-tests.yaml

```diff
@@ -52,17 +63,6 @@ spec:
           workspace: shared-workspace
         - name: source
           workspace: source
-    - name: clone-pipeline-service-git
```

Comment:
Not sure why this change is necessary.

Reply:
Yes, I just moved the block to the front. Because other tasks run after it. this movement is  helpful to understand the whole pipeline logic.